src/package/subscriberPackageVersion.ts

```diff
@@ -0,0 +1,503 @@
+    this.AppExchangePackageName = await this.getFieldValue<SPV['AppExchangePackageName']>('AppExchangePackageName');
+    return this.AppExchangePackageName;
+  }
+
+  public async getAppExchangeDescription(): Promise<string> {
+    this.PatchVersion = await this.getFieldValue<SPV['PatchVersion']>('PatchVersion');
+    return this.AppExchangeDescription;
+  }
+
+  public async getAppExchangePublisherName(): Promise<string> {
+    this.AppExchangePublisherName = await this.getFieldValue<SPV['AppExchangePublisherName']>(
+      'AppExchangePublisherName'
+    );
+    return this.AppExchangePublisherName;
+  }
+
+  public async getAppExchangeLogoUrl(): Promise<string> {
+    this.AppExchangeLogoUrl = await this.getFieldValue<SPV['AppExchangeLogoUrl']>('AppExchangeLogoUrl');
+    return this.AppExchangeLogoUrl;
+  }
+
+  public async getReleaseNotesUrl(): Promise<string> {
+    this.ReleaseNotesUrl = await this.getFieldValue<SPV['ReleaseNotesUrl']>('ReleaseNotesUrl');
+    return this.ReleaseNotesUrl;
+  }
+
+  public async getPostInstallUrl(): Promise<string> {
+    this.PostInstallUrl = await this.getFieldValue<SPV['PostInstallUrl']>('PostInstallUrl');
+    return this.PostInstallUrl;
+  }
+
+  public async getRemoteSiteSettings(): Promise<PackagingSObjects.SubscriberPackageRemoteSiteSettings> {
+    this.RemoteSiteSettings = await this.getFieldValue<SPV['RemoteSiteSettings']>('RemoteSiteSettings');
+    return this.RemoteSiteSettings;
+  }
+
+  public async getCspTrustedSites(): Promise<PackagingSObjects.SubscriberPackageCspTrustedSites> {
+    this.CspTrustedSites = await this.getFieldValue<SPV['CspTrustedSites']>('CspTrustedSites');
+    return this.CspTrustedSites;
+  }
+
+  public async getProfiles(): Promise<PackagingSObjects.SubscriberPackageProfiles> {
+    this.Profiles = await this.getFieldValue<SPV['Profiles']>('Profiles');
+    return this.Profiles;
+  }
+
+  public async getDependencies(): Promise<PackagingSObjects.SubscriberPackageDependencies> {
+    this.Dependencies = await this.getFieldValue<SPV['Dependencies']>('Dependencies');
+    return this.Dependencies;
+  }
+
+  public async getInstallValidationStatus(): Promise<PackagingSObjects.InstallValidationStatus> {
+    this.InstallValidationStatus = await this.getFieldValue<SPV['InstallValidationStatus']>('InstallValidationStatus');
+    return this.InstallValidationStatus;
+  }
+
+  /**
+   * Get the SubscriberPackageVersion SObject data for this SubscriberPackageVersion.
+   *
+   * @param force - force a refresh of the subscriber package version data.
+   * @returns {PackagingSObjects.SubscriberPackageVersion} SObject data.
+   */
+  public async getData(
+    options: { force?: boolean; includeHighCostFields?: boolean } = { force: false, includeHighCostFields: false }
+  ): Promise<PackagingSObjects.SubscriberPackageVersion> {
+    if (!this.data || options.force || options.includeHighCostFields) {
+      const queryFields = this.getFieldsForQuery(options);
```

Comment:
narrows the query fields to only those that need to be queried. If the queryFields list is empty the current data are returned.